plugins/filter_template.rs

```diff
@@ -0,0 +1,30 @@
+mod rpc;
```

Comment:
The file is called "filter_template.rs" This suggests that it's a template for filters written in Rust, while, in fact, it is a sample filter written in Rust. Calling it filter_sample.rs or filter_example.rs might be more appropriate.

Reply:
I think it might be a good idea to delete it for now, since we already have a sample filter, so I got rid of it until we have the one example working.